packages/owlbot-bootstrapper/common-container/cloudbuild-deploy-container.yaml

```
@@ -0,0 +1,36 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This build file will build and push the common-container in googleapis-bootstrapper
+# to container registry. There will be a trigger in owlbot-bootstrapper that
+# will build and push the image when any merges to main in repo-automation-bots occur.
+# This will ensure that the common-container we call will stay up-to-date.
+
+steps: 
+  - name: gcr.io/cloud-builders/docker
+    id: "build-docker"
+    waitFor: ["-"]
+    dir: packages/owlbot-bootstrapper
```

Looks like this needs an extra directory
```suggestion
    dir: packages/owlbot-bootstrapper/common-container
```

---

great catch, thank you!